js/components/graph/Graph.js

```diff
@@ -162,6 +165,19 @@ export default class Graph extends React.Component {
             edgesList.push(entry);
           }
         });
+
+        nodesList.forEach(function(node) {
```

Comment:
This is good, but generally prefer the new arrow function syntax, `node => { ...`